agent/hasalTask.py

```diff
@@ -19,6 +19,23 @@ class HasalTask(object):
     DEFAULT_AGENT_STATUS_DIR = os.path.join(os.getcwd(), "agent_status")
     DEFAULT_AGENT_JOB_STATUS = {'BEGIN': 'begin', 'FINISH': 'finish', 'EXCEPTION': 'exception'}
     DEFAULT_DATA_EXPIRE_DAY = 7
+    DEFAULT_CONFIG_DN = "configs"
+    DEFAULT_CONFIG_NAME = "default.json"
+    JENKINS_CONFIG_NAME = "jenkins.json"
+    DEFAULT_SUITE_NAME = "suite.txt"
+    JENKINS_SUITE_NAME = "jenkins_suite.txt"
+    DEFAULT_INDEX_CONFIG_NAME = "inputLatencyAnimationDctGenerator.json"
+
+    class Unbuffered:
```

Comment:
This class was written for output to `stdout` and file handle at the same time.
I'll suggest to add the comments for this class.